frontend/src/components/common/Link.tsx

```diff
@@ -76,11 +88,15 @@ function PureLink(props: React.PropsWithChildren<LinkProps | LinkObjectProps>) {
   return (
     <MuiLink
       component={RouterLink}
-      to={{
-        pathname: createRouteURL(routeName, params),
-        search,
-        state,
-      }}
+      to={
+        props.onClick
+          ? '#'
+          : {
+              pathname: createRouteURL(routeName, params),
+              search,
+              state,
+            }
```

Comment:
I am not seeing the URL being affected by this.

Reply:
slightly adjusted logic here, URL now updates and also the middle mouse click works (opens a full details page in a new page)